css-three-column-layout/styles.css

```diff
@@ -0,0 +1,39 @@
+* {
+  font-family: serif;
+  font-weight: normal;
+  color: white;
+}
+
+.container {
+  flex-flow: wrap;
+  display: flex;
+  max-width: 1000px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: space-around;
+}
+
+html {
+  background-color: #1996da;
+}
+
+.col1 {
+  flex-basis: calc(50% / 3);
+}
+
+.widpic {
+  max-height: auto;
+  width: 150px;
+  flex-basis: 20%;
```

Comment:
`width` & `flex-basis` are essentially the same thing (with some caveats). You should not be setting both. You should be using `flex-basis` to essentally set the `width`